tools/cdb2sql/cdb2sql.cpp

```diff
@@ -1379,7 +1414,8 @@ static int run_statement(const char *sql, int ntypes, int *types,
         } else if (printmode & DISP_TABS) {
             fprintf(out, "\n");
         } else if (printmode & DISP_GENSQL) {
-            fprintf(out, ");\n");
+            fprintf(out, ");");
```

Comment:
We should not need `;` anymore.
`fprintf(out, ")");`

Reply:
This is to keep the current behavior, we use newline as delimiter but we output ';\n' in gensql.